userguide/kaui/includes/tags.adoc

```diff
@@ -1 +1,127 @@
-// For the tags, it would make sense to touch on the 'system' tags versus user tags, the former (Uppercase) having a strong effect on the behavior of the system, while the latter (lowercase) having NO effect on the behavior of the system.
+=== Overview
+
+In Kaui, you can apply one or more tags to accounts and invoices, a tag being a single value. Tags in Kill Bill can be used to convey information about an object type. A tag can just provide information about an object (user tag) or it can actually affect how the object is handled in Kill Bill (control tag).
+
+This section explains how to attach tags to accounts and invoices. It also describes the steps for creating the tag definitions on which a tag is based.
+
+[NOTE]
+*Note:* You can use the API to attach tags to even more object types, such as payments and subscriptions. For more information, see the "Tags" section of the relevant object type section (for example, Payments) in the https://killbill.github.io/slate/[API Reference].
+
+==== Types of Tags
+
+Kill Bill has two types of tags:
+
+* *Control tags*&#8212;Also referred to as _system tags_, these tags modify the behavior of the system. In Kaui, you can view and attach these tags, but you cannot add, change, or delete the underlying <<Add Tag Definition, tag definition>>. Kill Bill includes several control tags; see the API documentation for https://killbill.github.io/slate/#tag[a list].
+
```

Comment:
Looks like the [Account Tags](https://killbill.github.io/slate/#account-tags) section lists all the tags. I will move this to the [Tag](https://killbill.github.io/slate/#tag) section in the slate docs.